contrib/pipelines/secadvisor/README.md

```diff
@@ -5,30 +5,104 @@ Implements the IBM Security Advisor pipeline
 
 ## Setup Skydive
 
-First build and run skydive:
+Follow instructions on http://skydive.network/documentation/build to install prerequisites and prepare your machine to build the skydive code.
+Build and run skydive:
 
 ```
 make
 cp etc/skydive.yml.default skydive.yml
-sudo $(which skydive)) allineone -c skydive.yml
+// adjust settings in skydive.yml
+sudo $(which skydive) allineone -c skydive.yml
 ```
 
+Alternatively, run a skydive agent on each of your hosts and a skydive analyzer on one host.
+
+On each machine:
+
+```
+sudo skydive agent -c skydive.yml
+```
+
+Be sure to set the field "analyzers" in the skydove.yml to point to the analyzer.
+
+On one machine:
+
+```
+sudo skydive analyzer -c skydive.yml
+```
+
+
 ## Setup Minio
 
-Install and run minio (an AWS S3-like object store).
+Follow the instructions on https://docs.min.io/docs/minio-quickstart-guide.html to install and run a minio object store (an AWS S3-like object store).
+
+```
+wget https://dl.min.io/server/minio/release/linux-amd64/minio
+chmod +x minio
+./minio server /data
+```
+
+This will run a basic minio object store server on your machine, placing the objects in "/data".
+Note the printed output in the logs from the ./minio command and take down the values of "Endpoint", "AccessKey", and "SecretKey". Place these items in the secadvisor.yml in the appropriate locations (see below). 
+Create a bucket in the object store to hold the security-advisor data.
+
+## Prepare secadvisor.yml
+
+```
+cp secadvisor.yml.default secadvisor.yml
+```
+
+Set the fields "endpoint", "access_key", "secret_key", "bucket", "cluster_net_masks", etc.
```

Comment:
@orozery Or can you work with @KalmanMeth to add some explanations on the authentication methods that are available and add explanations how to configure them. It will be very useful for users that want to use this feature. Maybe also some example for end-point (IBM-S3 and AWS-S3 ?)

(Maybe link to the relevant section in the secadvisor/secadvisor.yml.default) file